commit/gasworker/observation.go

```diff
@@ -0,0 +1,15 @@
+package gasworker
```

Comment:
Lets not call this package `gasworker`, I think `fees` is more general (not EVM specific) and the `Worker` aspect is just a type inside this package.